src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -414,7 +414,7 @@ private void tryGettingUnstuckFromLeaves()
      */
     private boolean plantSapling(@NotNull BlockPos location)
     {
-        if (BlockPosUtil.getBlock(world, location) != Blocks.AIR)
+        if (BlockPosUtil.getBlock(world, location) != Blocks.AIR && !(world.getBlockState(location).getBlock() instanceof BlockSapling))
```

Comment:
Lets save this block to a variable, so that we don't have to find it multiple times.